invenio_requests/assets/semantic-ui/js/invenio_requests/timeline/state/actions.js

```diff
@@ -43,6 +51,19 @@ export const fetchTimeline = (loadingState = true, chosenParams) => {
   };
 };
 
+export const setPage = (page, refresh = true) => {
```

Comment:
Looks like `refresh` param is never called?
```suggestion
export const setPage = (page) => {
```